lib/collibra/client/get_asset_by_id.ts

```diff
@@ -0,0 +1,11 @@
+import * as TE from "fp-ts/TaskEither"
+import * as E from "fp-ts/Either"
+import { AxiosInstance, AxiosResponse } from "axios"
+import { pipe } from "fp-ts/lib/function"
+
+export const getAssetByID = (client: Net.Client) => (id: string) =>
+  pipe(
+    TE.tryCatch(() => client.get<AxiosResponse<Collibra.Asset>>(`/assets/${id}`), E.toError),
+    TE.bindTo("res"),
```

Comment:
This one can also be removed. Bind is only needed when you want to cache an intermediate result to be used later in the pipeline